### ISSUE TITLE
Could not find a version of package yireo/magento2-webp2 

### ISSUE THREAD

**Issue by ecoprince** [yireo/Yireo_Webp2#109]
I am trying to install the extension with composer but I got the below error on the composer require command

**Command:** composer require yireo/magento2-webp2 

`[InvalidArgumentException]                                                                                                                                                         
  Could not find a version of package yireo/magento2-webp2 matching your minimum-stability (stable). Require it with an explicit version constraint allowing its desired stability.  `

I already checked the root composer.json and everything is good. I already installed many other third-party extensions recently on site. But I got an error on this extension only.

My composer.json contains a valid Magento repository and auth.json

    "repositories": {
            "0": {
                "type": "composer",
                "url": "https://repo.magento.com/"
            }
    }

    {
        "http-basic": {
            "repo.magento.com": {
                "username": "XXXXXXXXXXXXXXXXXXXXXXXXXX",
                "password": "XXXXXXXXXXXXXXXXXXXXXXXXXX"
            }
    }

Any suggestion?

**Comment by jissereitsma** [comment]
There's numerous releases available, each one with their own requirements: https://github.com/yireo/Yireo_Webp2/releases Each release its `composer.json` file can be inspected to check the requirements. What composer is doing is simply match those requirements with your environment. Logically, your environment is not matching. If you mention more specific things about your environment (Magento version, PHP version, etc) it probably would help.

**Comment by jissereitsma** [comment]
Also, your `minimum-stability` is set to `stable` while these packages start with 0, because there are still many changes coming up. Set the minimum stability to `dev` or something, and you can use any package.

**Comment by ecoprince** [comment]
Hi @jissereitsma, thank you for the quick response. I just changed  `minimum-stability` to `dev` from composer.json and re-try composer require command but no luck.

After further investigation, I found that we declared `"packagist.org": false` in composer.json because we use our private packagist repository to install any project-specific extensions.

I temporary removed `"packagist.org": false` and the extension was installed succesfully.
